endpoints/openrtb2/sample-requests/invalid-whole/alias-bidder-self.json

```diff
@@ -13,17 +13,19 @@
         },
         "ext": {
           "appnexus": {
-            "placementId": 12883451
+            "placementid": 12883451
```

Comment:
Is this a typo? Looks like it was changed from `placementId` to `placementid`.

Reply:
Typo. Corrected